.github/workflows/ascent.yml

```
@@ -0,0 +1,29 @@
+name: 🐧 In Situ Vis with Ascent
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.head_ref }}-insituvis
```

Either we make this a unique one
```suggestion
  group: ${{ github.ref }}-${{ github.head_ref }}-ascent
```

or even better - we put them into one `.yml` file (adding the new `jobs` key alongside the existing one :) )

With the current `concurrency` `group` name, both the `ascent.yml` and the `insitu.yml` would cancel each other.